.github/workflows/dev_pr/labeler.yml

```diff
@@ -45,6 +45,9 @@ lang-R:
 lang-ruby:
   - ruby/**/*
 
+lang-matlab:
```

Comment:
Could you put this between `lang-julia` and `lang-python`?
This list is sorted in alphabetically.